.helm/deploy-to-cluster.sh

```
@@ -1,111 +1,10 @@
-  done
-
-  values="$values --set apiCache.image.repository=docker.io/${docker_hub_account}/ecamp3-varnish"
-
-  values="$values --set postgresql.dbBackupRestoreImage.pullPolicy=$pull_policy"
-  values="$values --set postgresql.dbBackupRestoreImage.repository=docker.io/${docker_hub_account}/ecamp3-db-backup-restore"
+ACTION="${1:-diff}"
 
-  helm uninstall ecamp3-"$instance_name"-"$i" || true
-  helm upgrade --install ecamp3-"$instance_name"-"$i" $SCRIPT_DIR/ecamp3 $values
-done
+export NAME=$(jq -r '.NAME' $SCRIPT_DIR/ecamp3/env.yaml)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)